src/pages/workspaces/workspace/analysis/modals/ComputeModal.test.js

```diff
@@ -993,4 +997,26 @@ describe('ComputeModal', () => {
       }))
     })
   })
+
+  it('renders default cost estimate', async () => {
+    // Arrange
+    const expectedRuntimeConfigCost = 0.40
+    const expectedRuntimeConfigBaseCost = 0.15
+    const expectedPersistentDiskCostMonthly = 0.20
+
+    asMockedFn(runtimeConfigCost).mockReturnValue(expectedRuntimeConfigCost)
```

Comment:
just FYI, asMockedFn is really only helpful when used in Typescript land... no harm in using it here though.